Heatington/Models/FormatedResultHolder.cs

```diff
@@ -0,0 +1,20 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Heatington.Models;
+
+public class FormatedResultHolder(
+    DateTime startTime,
+    DateTime endTime,
+    double heatDemand,
+    double electricityPrice,
+    ProductionUnit boiler,
+    double netProductionCost)
+{
+    public DateTime StartTime { get; } = startTime;
+    public DateTime EndTime { get; } = endTime;
+    public double HeatDemand { get; } = heatDemand;
+    public double ElectricityPrice { get; } = electricityPrice;
+    public ProductionUnit Boilers { get; set; } = boiler;
+    public double NetProductionCost { get; set; } = netProductionCost;
+}
```

Comment:
I am not sure, but the FormattedResultHolder props will get set for only in the population phase?

If so consider using init?
https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/proposals/csharp-9.0/init
```suggestion

public class FormatedResultHolder(
    DateTime startTime,
    DateTime endTime,
    double heatDemand,
    double electricityPrice,
    ProductionUnit boiler,
    double netProductionCost)
{
    public DateTime StartTime { get; } = startTime;
    public DateTime EndTime { get; } = endTime;
    public double HeatDemand { get; } = heatDemand;
    public double ElectricityPrice { get; } = electricityPrice;
    public ProductionUnit Boilers { get; init; } = boiler;
    public double NetProductionCost { get; init; } = netProductionCost;
}
```